lib/wanda/results.ex

```diff
@@ -22,4 +25,24 @@ defmodule Wanda.Results do
     }
     |> Wanda.Repo.insert!()
   end
+
+  @doc """
+  Gets execution results from the database.
+  """
+  @spec get_execution_results(String.t(), number(), number()) :: [ExecutionResult.t()]
+  def get_execution_results(group_id, page, items_per_page \\ 10) do
```

Comment:
+1 for list_execution_results, as it is the common Phoenix name

Reply:
> I would change the page and items_per_page terminology. They are really frontend oriented namings.
Why not limit and offset (and I would calculate the offset in the controller code).

Not sure about this, limit and offset are really database related terms.
I like page and items per page because they are user facing names, the one you expect in a context.

> I liked this example: https://github.com/gothinkster/elixir-phoenix-realworld-example-app/blob/master/lib/real_world/blog/blog.ex#L22

we could use this idiom, I like it

Reply:
I agree with the guys, we might find a different idiom, how would the reference provided look like?
Something like this?
- `Results.list_execution_results()` -> full list 
- `Results.list_execution_results(filter_by: [group_id: some_id])` -> filtering
- `Results.list_execution_results(page: page, items_per_page: items_per_page)` -> paginating
- `Results.list_execution_results(filter_by: [group_id: some_id], page: page, items_per_page: items_per_page)` -> filtering + paginating